devices/product_manager.py

```diff
@@ -0,0 +1,154 @@
+"""
+Manage products in the shelf: insert and remove.
+"""
+import asyncio
+import logging
+import os
+from pathlib import Path
+from typing import List
+
+import aiofile
+import aiopubsub
+import boto3
+from boto3.dynamodb.conditions import Attr
+from botocore.exceptions import ClientError
+from marshmallow import Schema, fields, post_load
+from marshmallow.exceptions import ValidationError
+from models.product import Product, ProductSchema, ProductTag
+
+
+class ProductShelf:
+    #pylint: disable=too-few-public-methods
+    """TODO"""
+    def __init__(self, products: List[Product]):
+        self.products = products
+
+class ProductShelfSchema(Schema):
+    """TODO"""
+    products = fields.List(fields.Nested(ProductSchema))
+
+    @post_load
+    def make_user(self, data, **kwargs):
+        #pylint: disable=missing-function-docstring,unused-argument,no-self-use
+        return ProductShelf(**data)
+
+
+class ProductManager:
+    #pylint: disable=too-many-instance-attributes
+    """
+    TODO.
```

Comment:
This TODO is still necessary ? Also at line 22, 27 and 66

Reply:
Yes is the documentation of the class.